app/views/lectures/edit/_form.html.erb

```diff
@@ -1,63 +1,191 @@
+        <div class="container small-width lecture-pane">
+          <%= render partial: 'lectures/edit/people',
+              locals: { lecture: lecture } %>
+          <%= render partial: 'lectures/edit/tutorials',
+              locals: { lecture: lecture } %>
+        </div>
+      </div>
+
+      <!-- Organisation -->
+      <div class="tab-pane fade" id="lecture-pane-orga"
+          role="tabpanel" aria-labelledby="lecture-nav-orga" tabindex="0">
+        <div class="container small-width lecture-pane">
+          <%= render partial: 'lectures/edit/organizational_concept',
+              locals: { lecture: lecture } %>
+        </div>
+      </div>
+
+      <!-- Communication -->
+      <div class="tab-pane fade" id="lecture-pane-communication"
+          role="tabpanel" aria-labelledby="lecture-nav-communication" tabindex="0">
+        <div class="container small-width lecture-pane">
+          <%= render partial: 'lectures/edit/announcements',
+              locals: { lecture: lecture, announcements: announcements } %>
+          <%= render partial: 'lectures/edit/forum',
+              locals: { lecture: lecture } %>
+          <%= render partial: 'lectures/edit/comments',
+              locals: { lecture: lecture } %>
+        </div>
+      </div>
+
+      <!-- Homework -->
+      <div class="tab-pane fade" id="lecture-pane-homework"
+          role="tabpanel" aria-labelledby="lecture-nav-homework" tabindex="0">
+        <div class="container small-width lecture-pane">
+          <%= render partial: 'lectures/edit/assignments',
+              locals: { lecture: lecture } %>
+        </div>
+      </div>
+
+      <!-- Info -->
+      <div class="tab-pane fade" id="lecture-pane-info"
+          role="tabpanel" aria-labelledby="lecture-nav-info" tabindex="0">
+        <div class="container small-width lecture-pane">
+          <h3 class="lecture-pane-header">
+            <%= t('basics.course_editors') %>
+              <%= helpdesk(t('admin.lecture.info.course_editors'), false) %>
+          </h3>
+          
+          <div class="col-4">
+            <% if lecture.course.editors.present? %>
+              <ul>
+                <% lecture.course.editors.each do |e| %>
+                  <li>
+                    <%= e.name %>
+                  </li>
+                  <% end %>
+              </ul>
+              <% else %>
+                <br>
+                <%= t('admin.lecture.no_course_editors') %>
+                  <% end %>
+          </div>
+
+          <%= render partial: 'lectures/edit/tags',
+              locals: { lecture: lecture } %>
+          <%= render partial: 'lectures/edit/imported_media',
+              locals: { lecture: lecture } %>
+          <%= render partial: 'lectures/edit/erdbeere',
+              locals: { lecture: lecture } %>
+        </div>
       </div>
     </div>
   </div>
+
+
+  <!-- TODO (Lectures): Find out when these partials are used -->
```

Comment:
we may have to talk about these partials here

Reply:
I went through all modals in the UI and they still work fine. Exception to this are the special cases for seminars, I did not go through them in the UI. However, everything should work as beforehand since this PR just changes the UI representation. All elements have been on the same page beforehand (in the accordions) and now still are on one page (with a tab navigation bar), so data loading still stays the same.

I wondered where the `tags/modal` is used and have not found a button to activate on the lectures edit page.

Reply:
We came to the conclusion that this modal is not used here anymore, so I deleted the respective line in 6ea1e758